config/environments/production.rb

```diff
@@ -83,7 +83,7 @@
 
   # Use a different logger for distributed setups.
   # require 'syslog/logger'
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
+  config.logger = GELF::Logger.new("localhost", 12219, "WAN", { :facility => "YOUR_APP_NAME" })
```

Comment:
Here is missing the exchange of "YOUR_APP_NAME". Isn't there a environment variable with the app name, so that can be used here instead of a plain string?